conf/locale/eo/LC_MESSAGES/django.po

```diff
@@ -5045,6 +5104,18 @@ msgstr "Sérvér érrör. Pléäsé trý ägäïn ïn ä féw mïnütés. Ⱡ'σ
 msgid "No endpoint was provided for EdxNotes."
 msgstr "Nö éndpöïnt wäs prövïdéd för ÉdxNötés. Ⱡ'σяєм ιρѕ#"
 
+#: lms/djangoapps/instructor/tests/test_api.py
+#: lms/djangoapps/instructor/views/api.py
+msgid "This student (%s) will skip the entrance exam."
```

Comment:
Where do you define this string? Please use NAMED variables, not "%s". See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#general-internationalization-rules